invenio_damap/assets/semantic-ui/js/invenio_damap/DMPAuthButton.js

```diff
@@ -0,0 +1,55 @@
+// This file is part of Invenio-DAMAP
+// Copyright (C) 2023-2024 Graz University of Technology.
+// Copyright (C) 2023-2024 TU Wien.
+//
+// Invenio-DAMAP is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Button, Icon, Popup } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
```

Comment:
```suggestion
import { i18next } from "@translations/invenio_curations/i18next";
```

This requires to specify a new webpack entry point and initialize the translations. Otherwise, it will lookup the translations from the `invenio_app_rdm` package

Reply:
Same as above, I reverted everything concerning translations.